omero/developers/Model.txt

```diff
@@ -567,7 +563,7 @@ multiple threads, but simply due to stale state.
 
 .. note:: 
     There is an issue with multiple users in which a 
-    ``SecurityViolation`` is thrown instead of an ``OptimisticLockException``. 
+    ``SecurityViolation`` is thrown instead of an ``OptimisticLockException``.
     See :ticket:`1649` for more information.
```

Comment:
remove mentions of this ticket, see http://trac.openmicroscopy.org.uk/ome/ticket/1649#comment:13